.github/workflows/continuous-integration-workflow.yml

```diff
@@ -12,30 +12,27 @@ jobs:
       matrix:
         distro: ['fedora:latest', 'fedora:rawhide', 'ubuntu:latest']
         cxx: ['g++', 'clang++']
+        cxx_extra_flags: ['']
         cmake_build_type: ['Release', 'Debug']
         backend: ['OPENMP']
         clang-tidy: ['']
         include:
           - distro: 'fedora:intel'
             cxx: 'icpc'
+            cxx_extra_flags: '-diag-disable=177,10441'
             cmake_build_type: 'Release'
-            backend: 'OPENMP'
```

Comment:
We can drop that https://github.com/kokkos/kokkos/actions/runs/3961405441/jobs/6786791382#step:11:9 
```
    -DKokkos_ENABLE_HWLOC=ON \
    -DKokkos_ENABLE_=ON \
    -DKokkos_ENABLE_TESTS=ON \
```
empty backend leads to `Kokkos_ENABLE_=ON`.